mods/d2k/rules/structures.yaml

```diff
@@ -79,7 +80,8 @@ construction_yard:
 		FactionImages:
 			atreides: conyard.atreides
 			ordos: conyard.ordos
-			corrino: conyard.corrino
+			smuggler: conyard.ordos
+			mercenaries: conyard.ordos
```

Comment:
You could make these definitions more efficient by making the default `Image: conyard.ordos` (and so on for other buildings), and then you only need to override `harkonnen: conyard.harkonnen` instead of overriding ordos, smugger, and mercinary.